net-print/fax4cups/fax4cups-1.29-r1.ebuild

```diff
@@ -24,14 +25,13 @@ RDEPEND="${DEPEND}
 	app-admin/sudo
 "
 
-S=${WORKDIR}/fax4CUPS-${PV}
-
 src_install() {
 	doman fax4CUPS.1
 
 	exeinto $(cups-config --serverbin)/backend
```

Comment:
Annoyingly, this means we need cups in BDEPEND too. And a || die inside of the subshell.